### ISSUE TITLE
Problems opening the select box when navigation in VoiceOver using control + option + shift + arrow

### ISSUE THREAD

**Issue by HeleneRiddervold** [sagalbot/vue-select#1297]
When I use the tab-key to navigate to the element in VoiceOver with Safari, the component works as expected. But I often navigate element by element using control + option + shift + arrow. When I do that, I cannot open the select/combobox. Using control + option + shift, or shift, or enter, or arrows wont work. This is vital, since many users of screen readers use this approach when navigating complex forms with a lot of titles and additional information.

**Desktop :**
 - OS: iOS
 - Browser: safari
 - Version 14.0
